qa/common/util.sh

```diff
@@ -22,7 +22,7 @@ ANALYZER_LOG=${ANALYZER_LOG:="$LOGS_DIR/test.log"}
 
 mkdir -p $LOGS_DIR
 
-create_logs_dir() {
```

Comment:
Why was this change made?

Reply:
It seems that I forgot to include the "function" keyword when I implemented the function. I have now added it.